corenlp/src/main/scala/org/clulab/processors/coserver/ProcessorActor.scala

```diff
@@ -0,0 +1,186 @@
+package org.clulab.processors.coserver
```

Comment:
Forgive me for being dense, but what is `coserver` supposed to suggest (specifically the "co")?